tests/utils/test_sentry.py

```diff
@@ -2,10 +2,13 @@
 
 from unittest.mock import patch
 
+import pytest
+
 from supervisor.bootstrap import initialize_coresys
 
 
-async def test_sentry_disabled_by_default(supervisor_name):
+@pytest.mark.usefixtures("supervisor_name", "docker")
```

Comment:
This is unrelated to the PR, this test is just broken in devcontainer right now. Calling this fixture keeps it working there.

However on the bright side, it did find more blocking I/O. Turns out initializing the docker `APIClient` object does blocking I/O. Normally this part of setup is all mocked because we can't do that on devcontainer but running this test un-mocked found it 😆